code401challenges/src/main/java/utilities/FizzBuzzTree.java

```diff
@@ -0,0 +1,38 @@
+package utilities;
+
+import tree.*;
+
+
+public class FizzBuzzTree<E>
+{
+    private static void FizzBuzzTree(Node<Object> node)
+    {
+        if(node != null)
+        {
+            FizzBuzzTree(node.left_Node);
+            FizzBuzzTree(node.right_Node);
+
+            if ((Integer)node.node_Data % 15 == 0)
+            {
+                node.node_Data = "FizzBuzz";
+            }
+            else if ((Integer) node.node_Data % 5 == 0)
+            {
+                node.node_Data = "Buzz";
+            }
+            else if ((Integer) node.node_Data % 3 == 0)
+            {
+                node.node_Data = "Fizz";
+            }
+        }
+    }
```

Comment:
👍 This will preform a post-order traversal since it recurses on the left and right before performing the fizz buzz operation.